.github/workflows/ci.yml

```diff
@@ -45,14 +45,15 @@ jobs:
           cd sdk/swift/TeaclaveClientSDK && xcodebuild -scheme TeaclaveClientSDK
   format:
     runs-on: ubuntu-20.04
-    container: teaclave/teaclave-build-ubuntu-1804-sgx-2.14:0.1.5
+    container: ghcr.io/henrysun007/teaclave-build-ubuntu-2004-sgx-2.17.1:upgrade
```

Comment:
I'll add you to the `teaclave` org in DockerHub. Let's use this org to host the image.

Reply:
I was added to `teaclave` organization in docker Hub, but seem to have no permission to create new repos.

Reply:
I have invited you as the admin. Can you try again? Thanks!

Reply:
I have added you as the owner. Please try again.

Reply:
The pushing succeeded this time. Thanks, mingshen.